ops/services/okta-global/main.tf

```diff
@@ -53,3 +53,53 @@ resource "okta_policy_rule_signon" "app_mfa" {
   session_idle       = 720
   session_lifetime   = 720
 }
+
+//Changes made here also need to be made to the Azure WAF!
+resource "okta_network_zone" "sr_network_zone" {
+  name  = "Geographic Restrictions"
+  type  = "DYNAMIC"
+  usage = "BLOCKLIST"
+  dynamic_locations = [
```

Comment:
Is this array the same as the array in the Azure WAF? Would it be possible to turn this into a local variable?